### PR TITLE
feat: add can_pay_by_paytm to event model and schema and refactor other keys

### LINKED ISSUE #6340: Add paytm variables to event model
**Is your feature request related to a problem? Please describe.**

Parent issue - https://github.com/fossasia/open-event-server/issues/6247
As we are moving forward with paytm now, we should have variables to ask organizer about their info if they want to link paytm to their event as a payment gateway

**Describe the solution you'd like**

I'll add new fields required for paytm to event model

**Additional context**

On it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #6340 

#### Short description of what this resolves:
Adds `can_pay_by_paytm` to event model and schema

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: migrations/versions/rev-2019-08-12-13:13:21-24e068e57321_.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = '24e068e57321'
+down_revision = '246b5b6123af'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('can_pay_by_paytm', sa.Boolean(), nullable=True))
```
Comment (mrsaicharan1) [inline migrations/versions/rev-2019-08-12-13:13:21-24e068e57321_.py]: nullable = False.

Comment (shreyanshdwivedi) [reply migrations/versions/rev-2019-08-12-13:13:21-24e068e57321_.py]: Just following constraints of other columns to maintain consistency.

Comment (kushthedude) [inline migrations/versions/rev-2019-08-12-13:13:21-24e068e57321_.py]: Add `server_default`

Comment (shreyanshdwivedi) [reply migrations/versions/rev-2019-08-12-13:13:21-24e068e57321_.py]: server_default is compulsory when nullable is False
And moreover like I mentioned above I'm trying to keep all the similar fields consistent that's why didn't add it

Comment (iamareebjamal) [reply migrations/versions/rev-2019-08-12-13:13:21-24e068e57321_.py]: It should be nullable false

File: app/api/schema/events.py [hunk 2]
```diff
@@ -91,6 +91,7 @@ def validate_timezone(self, data, original_data):
     can_pay_onsite = fields.Bool(default=False)
     can_pay_by_omise = fields.Bool(default=False)
     can_pay_by_alipay = fields.Bool(default=False)
+    can_pay_by_paytm = fields.Bool(allow_none=False, default=False)
```
Comment (iamareebjamal) [inline app/api/schema/events.py]: allow_none is not needed

File: app/models/event.py [hunk 3]
```diff
@@ -106,6 +106,7 @@ class Event(SoftDeletionModel):
     can_pay_onsite = db.Column(db.Boolean, default=False)
     can_pay_by_omise = db.Column(db.Boolean, default=False)
     can_pay_by_alipay = db.Column(db.Boolean, default=False)
+    can_pay_by_paytm = db.Column(db.Boolean, default=False, nullable=False)
```
Comment (uds5501) [inline app/models/event.py]: `nullable=False` can be skipped

Comment (iamareebjamal) [reply app/models/event.py]: No, why?

Comment (uds5501) [reply app/models/event.py]: @iamareebjamal Okay wait, you are right. My bad.
Just as a suggestion, should the `can_pay_by_omise` and similar fields also have this condition added to them?

Comment (iamareebjamal) [reply app/models/event.py]: Yes

**Comment by shreyanshdwivedi** [comment]
@uds5501 @mrsaicharan1 @kushthedude please review

**Comment by shreyanshdwivedi** [comment]
@iamareebjamal @mrsaicharan1 @uds5501 updated the down_revision and revises
Please take a look

**Pull request merged**